bundles/Planning/Resources/assets/planning.js

```diff
@@ -0,0 +1,304 @@
+import Vue from 'vue';
+import axios from 'axios';
+
+require('./planning.css');
+
+(function () {
+    const currentUrl = window.location.href;
+
+    const app = new Vue({
+        el: '#app',
+        data: {
+            hideUsers: {},
+            toggle: {},
+            sprints: [],
+            users: {},
+            projects: {},
+            numberLoaded: 0,
+            projectFilter: '',
+            userFilter: ''
+        },
+        computed: {
+            sortedUsers: function () {
+                if (this.users === {}) {
+                    return [];
+                }
+
+                var arr = Object.keys(this.users).map(function (i) {
+                    return this.users[i];
+                }.bind(this));
+
+                arr = arr.filter(function (item) {
+                    return item.displayName.toLowerCase()
+                        .indexOf(this.userFilter) !== -1;
+                }.bind(this));
+
+                arr = arr.sort(function (a, b) {
+                    if (a.key === 'unassigned') {
+                        return 1;
+                    }
+                    return (a.displayName.toLocaleLowerCase() > b.displayName.toLocaleLowerCase()) ? 1 : -1;
+                });
+
+                return arr;
+            },
+            sortedProjects: function () {
+                if (this.projects === {}) {
+                    return [];
+                }
+
+                var arr = Object.keys(this.projects).map(function (i) {
+                    return this.projects[i];
+                }.bind(this));
+
+                arr = arr.filter(function (item) {
+                    return item.name.toLowerCase()
+                        .indexOf(this.projectFilter) !== -1;
+                }.bind(this));
+
+                arr = arr.sort(function (a, b) {
+                    return (a.name.toLocaleLowerCase() > b.name.toLocaleLowerCase()) ? 1 : -1;
+                });
+
+                return arr;
+            }
+        },
+        created: function () {
+            var hideUsers = localStorage.getItem('hideUsers');
+
+            if (hideUsers !== null) {
```

Comment:
Yoda style must we use?

Reply:
Should be handled by coding standards tools